projects/ui-components/lf-login/login-utils/lf-login.service.ts

```diff
@@ -79,8 +80,22 @@ export class LfLoginService {
       this.exchangeCodeForToken_lock = true;
       this.code_verifier = localStorage.getItem(this.codeVerifierStorageKey)!;
       if (callBackURIParams.authorizationCode && this.code_verifier) {
-        const response = await this.getTokenAsync(callBackURIParams.authorizationCode);
-        await this.parseTokenResponseAsync(response);
+        try {
+          const response = await this.tokenClient?.getAccessTokenFromCode(callBackURIParams.authorizationCode, this.redirect_uri, this.client_id, undefined, this.code_verifier);
```

Comment:
same comment as the other response!
If we aren't sure the tokenClient will be there, can we be sure the response will be?